tui/tests/test_snapshots.py

```
@@ -0,0 +1,77 @@
+    The Feeds tab refreshes every 3 seconds and includes a UTC timestamp plus
+    an incrementing refresh counter, both of which cause spurious snapshot
+    mismatches.  This callback pins the timer to a fixed string.
+    """
+    try:
+        timer = pilot.app.query_one("#timer-display", Static)
+        timer.update("Last refresh: 00:00:00 UTC | Next in 3s | Refresh #1")
+    except NoMatches:
+        pass  # Tab not visible / widget not yet mounted — safe to skip
+
+
+async def _stabilise_insights(pilot) -> None:
```

Ruff (0.15.4)

[warning] Unused function argument: `pilot`

(ARG001)